explorer/frontend/utils/common.js

```diff
@@ -1,5 +1,20 @@
 import { TRANSACTION_CHART_TYPE } from '@/constants';
 
+/**
+ * Checks if the value is numeric.
+ * @param {any} value - Value.
+ * @returns {boolean} True if the value is numeric, false otherwise.
+ * @example
+ * isNumeric(123); // true
+ * isNumeric('123'); // true
+ * isNumeric('abc'); // false
+ * isNumeric(null); // false
+ * isNumeric(undefined); // false
+ * isNumeric(NaN); // false
+ * isNumeric(Infinity); // false
+ */
+const isNumeric = value => (typeof value === 'number' || typeof value === 'string') && isFinite(value);
+
```

Comment:
What about isFinite('')? is this valid? 🤔

Reply:
@Wayonb it returns `true` 😬. Thanks, fixed 👍.